module1/homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,24 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    
+    if( (sequence == 0) || (sequence == 1) ){
+    return sequence;
+    }
+    
+    int a = 1; 
+    int b = 1;
+    
+    for(int i = 0; i < sequence - 2; i++) {
+        std::swap(a, b);
+        b += a;
+    }
+    return b;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if (sequence == 0 || sequence == 1){
+        return sequence;
+        }
```

Comment:
```suggestion
    if (sequence == 0 || sequence == 1) {
        return sequence;
    }
```
Formatting, indentation